docs/content/faq/how-can-i-deploy-a-docker-image.md

```diff
@@ -0,0 +1,43 @@
+# How can I deploy a Docker image to Flynn?
+
+Flynn supports running Docker containers from Docker registries. Note that currently the volumes of Docker apps are ephemeral and will be wiped when the instance reboots or is scaled down.
+
+Start by creating a config file for your Docker app, specifying the command to run, the job type, the init command, and any open ports.
+
+    {
+      "processes": {
+        "server": {
+          "cmd": ["memcached", "-u", "nobody", "-l", "0.0.0.0", "-p", "11211", "-v"],
+          "data": true,
+          "ports": [{
+            "port": 11211,
+            "proto": "tcp",
+            "service": {
+              "name": "memcached",
+              "create": true,
+              "check": { "type": "tcp" }
+            }
+          }]
+        }
+      }
+    }
+
+
+Flynn requires the Docker container's Docker registry url to install. The easiest way to get this is to install the container from the Docker registries, then run `docker images` to get the ID.
```

Comment:
`The easiest way to get this is to pull the image from the registry and print it's ID using the Docker CLI`